js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadRoot.tsx

```diff
@@ -48,6 +48,44 @@ export const AssetLaunchpad = ({
   );
 };
 
+export const BackfillLaunchpad = ({
+  repoAddress,
+  sessionPresets,
+  assetJobName,
+  open,
+  setOpen,
+  onSaveConfig,
+}: {
+  repoAddress: RepoAddress;
+  sessionPresets?: Partial<IExecutionSession>;
+  assetJobName: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+  onSaveConfig: (config: any) => void;
+}) => {
+  const title = 'Config Editor';
+
+  return (
+    <Dialog
+      style={{height: '90vh', width: '80%'}}
+      isOpen={open}
+      canEscapeKeyClose={false}
+      canOutsideClickClose={true}
+      onClose={() => setOpen(false)}
+    >
+      <DialogHeader icon="layers" label={title} />
+      <CodeMirrorInDialogStyle />
+      <LaunchpadAllowedRoot
+        launchpadType="asset"
+        pipelinePath={assetJobName}
+        repoAddress={repoAddress}
+        sessionPresets={sessionPresets}
+        onSaveConfig={onSaveConfig}
+      />
```

Comment:
I don't think we want to render the entire root component within the dialog. Generally anything `Root` is an actual route component, intended to be rendered as an entire page and in conjunction with a URL path that maps to it. I think you probably just want to render whatever `LaunchpadAllowedRoot` is rendering, i.e. the `LaunchpadTransientSessionContainer` that it renders for the `asset` case.

Reply:
makes sense -- looks the main things that the LaunchpadAllowedRoot was doing there were querying for the execution params and pulling previously-saved config from local storage. Added code for the first bit, and the second bit is something we probably explicitly want to avoid doing 👍